src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.FSharp.targets

```diff
@@ -0,0 +1,35 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.FSharp.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(BuildingUnderFSharpSdk)' != 'true' and '$(IsCrossTargetingBuild)' == 'true' ">
+    <ImportByWildcardAfterMicrosoftCommonTargets>false</ImportByWildcardAfterMicrosoftCommonTargets>
+  </PropertyGroup>
+
+  <!-- Shim to select the correct Microsoft.NET.Sdk.FSharp.targets file.   
+       If running under desktop select Microsoft.sFSharp.targets file from VS deployment, 
+       if running core msbuild select Microsoft.FSharp.targets from dotnet cli deployment -->
+  <PropertyGroup>
+     <FSharpTargetsShim Condition = " '$(FSharpTargetsShim)' == '' and Exists('$(MSBuildToolsPath)\FSharp\Microsoft.FSharp.NetSdk.targets') ">$(MSBuildToolsPath)\FSharp\Microsoft.FSharp.NetSdk.targets</FSharpTargetsShim>
+     <FSharpTargetsShim Condition = " '$(FSharpTargetsShim)' == '' and Exists('$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\FSharp\Microsoft.FSharp.NetSdk.targets') ">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\FSharp\Microsoft.FSharp.NetSdk.targets</FSharpTargetsShim>
+  </PropertyGroup>
+  <Import Condition=" '$(BuildingUnderFSharpSdk)' != 'true' and Exists('$(FSharpTargetsShim)') and '$(BuildingUnderFSharpSdk)' != 'true' " Project="$(FSharpTargetsShim)" />
+
+  <Import Project="$(MSBuildToolsPath)\Microsoft.Common.CrossTargeting.targets"                            Condition=" '$(BuildingUnderFSharpSdk)' != 'true' and '$(IsCrossTargetingBuild)' == 'true' " />
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.targets\ImportAfter\*" Condition=" '$(BuildingUnderFSharpSdk)' != 'true' and '$(IsCrossTargetingBuild)' == 'true' and Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.targets\ImportAfter') "/>
```

Comment:
This isn't right either. In cross-targeting case, anything serving as LanguageTargets needs to import Microsoft.Common.CrossTargeting.targets *instead of* Microsoft.Common.targets (not in addition to it). When we're in the "outer build", there are supposed to be very few targets: (basically just build, rebuild, clean) that know how to loop over each target framework and call the "inner build", where the usual cast of targets (CoreCompile and friends) can continue to deal with just one target framework. This is overriding the inner build targets with the outer build targets, but leaving a bunch of inner build targets dangling that would do incorrect things if they were ever invoked.

In the cross-targeting case, this is all that VB and C# language targets do:

https://github.com/Microsoft/msbuild/blob/master/src/Tasks/Microsoft.CSharp.CrossTargeting.targets
https://github.com/Microsoft/msbuild/blob/master/src/Tasks/Microsoft.VisualBasic.CrossTargeting.targets